pkg/controllers/dynakube/injection/reconciler.go

```diff
@@ -77,6 +77,15 @@ func (r *reconciler) Reconcile(ctx context.Context) error {
 		return err
 	}
 
+	// do istio reconciliation for CodeModules here to enable cleanup of conditions
+	if r.istioReconciler != nil {
+		err = r.istioReconciler.ReconcileCodeModuleCommunicationHosts(ctx, r.dynakube)
+
+		if err != nil {
+			log.Info("error reconciling istio configuration for codemodules", "error", err)
```

Comment:
```suggestion
			log.Error("error reconciling istio configuration for codemodules", "error", err)
```
may be instead?

Reply:
fixed in c4c2371a

Reply:
this commit hash about different change.

Reply:
sorry my bad 76f4aaa0